config/rbac/role.yaml

```diff
@@ -116,6 +116,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - '*'
```

Comment:
This seems very permissive. Can it be restricted to only operator conditions?

Reply:
updated